src/subdomains/generic/user/models/kyc/kyc.service.ts

```diff
@@ -89,8 +90,22 @@ export class KycService {
     await this.spiderSyncService.syncKycUser(userId, true);
   }
 
+  async getKycCountries(userId: number): Promise<Country[]> {
+    const user = await this.userRepo.findOne({ where: { id: userId }, relations: ['userData'] });
+
+    return await this.countryService.getCountriesByKycType(user.userData.kycType);
+  }
+
+  async getKycCountriesByCode(code: string): Promise<Country[]> {
+    const userData = await this.getUserByKycCode(code);
+
+    return await this.countryService.getCountriesByKycType(userData.kycType);
+  }
```

Comment:
Combine these methods and use `this.getUser` to get the user (similar to `updateKycData`)